### PR TITLE
An initial gen capabilities list

### PR DESCRIPTION
**Pull request by jmlarson1**

### CODE REVIEW

**Comment by jlnav** [comment]
Interesting, an explicit generator capabilities list should help draw in users.

I'm wondering if other parts of the README or docs could have parts appended/restructured to emphasize unique workflow capabilities (e.g. sims&gens paradigm) just as clearly.

**Comment by shuds13** [comment]
@jmlarson1 
Maybe put a link to https://libensemble.readthedocs.io/en/docs-capabilities_section/examples/examples_index.html at the bottom?
If we want some example user codes, maybe https://warpx.readthedocs.io/en/latest/usage/workflows/libensemble.html?highlight=libensemble

File: README.rst [hunk 1]
```diff
@@ -244,18 +244,27 @@ libEnsemble generation capabilites include:
 - Tasmanian_ Toolkit for Adaptive Stochastic Modeling and Non-Intrusive ApproximatioN
 - VTMOP_ Fortran package for large-scale multiobjective multidisciplinary design optimization 
 
+libEnsemble has also coordinate many computational expensive simulations. Select examples include:
```
Comment (shuds13) [inline README.rst:247]: Grammer?

File: README.rst [hunk 2]
```diff
@@ -244,18 +244,27 @@ libEnsemble generation capabilites include:
...
+
+- OPAL_ Object Oriented Parallel Accelerator Library. (See this `IPAC manuscript`_.)
+- WarpX_ Advanced electromagnetic particle-in-cell code. (See example `WarpX + libE scripts`_.) 
+
+See a complete list of `example user scripts`.
```
Comment (shuds13) [inline README.rst:252]: link?

**Pull request merged**